artist/field/kinematic_rigid_body.py

```diff
@@ -141,33 +145,29 @@ def align(
         assert (
             len(self.actuators.actuator_list) == 2
         ), "The rigid body kinematic requires exactly two actuators, please check the configuration!"
```

Comment:
```suggestion
        if len(self.actuators.actuator_list) != 2:
            raise ValueError(f"The rigid body kinematic requires exactly two actuators but {len(self.actuators.actuator_list)} were specified, please check the configuration!")
```
I'd suggest to replace all asserts with proper exceptions as it is in principle possible to turn off asserts. However, this would also require expanding the tests to also cover each exception.

Reply:
Okay, I changed this part of the code to follow your suggestion.
I also opened an issue to fix all other parts of artist where we are using assert statements. 

Maybe this does not need to be solved right now in this branch?

Reply:
I pushed another version where i exchanged all assterts to ValueErrors except in the tests. I did not yet create all test cases.

Reply:
This breaks some tests. I will fix it

Reply:
The pytests work again now. However the code coverage was lowered due to the if statements, This makes the git checks fail. I dont know what to do about that.